physics/GFS_rrtmg_pre.F90

```diff
@@ -29,7 +29,7 @@ subroutine GFS_rrtmg_pre_run (im, levs, lm, lmk, lmp, n_var_lndp,        &
         prsi, prsl, prslk, tgrs, sfc_wts, mg_cld, effrr_in, pert_clds,         &
         sppt_wts, sppt_amp, cnvw_in, cnvc_in, qgrs, aer_nm, dx, icloud,        & !inputs from here and above
         coszen, coszdg, effrl_inout, effri_inout, effrs_inout,                 &
-        clouds1, clouds2, clouds3, clouds4, clouds5,                           & !in/out from here and above
+        clouds1, clouds2, clouds3, clouds4, clouds5,qci_conv,                  & !in/out from here and above
```

Comment:
Please mind the whitespace.

Reply:
What do you mean by mind the whitespace? Thanks

Reply:
```suggestion
        clouds1, clouds2, clouds3, clouds4, clouds5, qci_conv,                  & !in/out from here and above
```

Reply:
Ah, thanks. I was looking at every other type of whitespace ...